AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -69,41 +20,5 @@ public class AmplifyAWSServiceConfiguration {
     /// - Tag: AmplifyAWSServiceConfiguration.platformName
     public static let platformName = "amplify-swift"
 
-    /// Returns basic amount of metadata that includes both
-    /// [AmplifyAWSServiceConfiguration.amplifyVersion](x-source-tag://AmplifyAWSServiceConfiguration.amplifyVersion)
-    /// and
-    /// [AmplifyAWSServiceConfiguration.platformName](x-source-tag://AmplifyAWSServiceConfiguration.platformName)
-    /// in addition to the operating system version if `includesOS` is set to `true`.
-    ///
-    /// - Tag: AmplifyAWSServiceConfiguration.frameworkMetaDataWithOS
-    public static func frameworkMetaData(includeOS: Bool = false) -> FrameworkMetadata {
-        let osKey = "os"
-        guard let flutterVersion = platformMapping[Platform.flutter] else {
-            if includeOS {
-                return FrameworkMetadata(
-                    name: platformName,
-                    version: amplifyVersion,
-                    extras: [osKey: frameworkOS()]
-                )
-            }
-            return FrameworkMetadata(name: platformName, version: amplifyVersion)
-        }
-        var extras = [platformName: amplifyVersion]
-        if includeOS {
-            extras[osKey] = frameworkOS()
-        }
-        return FrameworkMetadata(name: Platform.flutter.rawValue,
-                                 version: flutterVersion,
-                                 extras: extras)
-    }
-
-    private static func frameworkOS() -> String {
-        // Please note that because the value returned by this function will be
-        // sanitized by FrameworkMetadata by removing anything not in a special
-        // character set that does NOT include the forward slash (/), the
-        // backslash (\) is used as a separator instead.
-        let separator = "\\"
-        let operatingSystem = DeviceInfo.current.operatingSystem
-        return [operatingSystem.name, operatingSystem.version].joined(separator: separator)
-    }
+    public static let userAgentLib: String = "lib/\(platformName)#\(amplifyVersion)"
```

Comment:
it looks like this implementation removes the option to include OS details in the user agent, just confirming that this is intended and won't break any previous or existing metrics.

Reply:
That's right. OS metadata was [added](https://github.com/awslabs/aws-sdk-swift/pull/1089) in the Swift SDK in 0.24.0

Thanks for calling it out though - I'll add this to the list of changes in the base PR description.